test/web-platform.js

```diff
@@ -54,3 +54,40 @@ describe("To-upstream tests", () => {
     it("Parsing: <" + expected.input + "> against <" + expected.base + ">", testURL(expected));
   }
 });
+
+describe("MDN examples", () => {
+  it("Checking all examples on MDN pass", () => {
+    const a = new URL("/", "https://developer.mozilla.org");
+    assert.strictEqual(a.href, "https://developer.mozilla.org/", "a");
```

Comment:
They don't help me as a developer at all. They cause the assertion message to be "a" instead of "asdfasdf did not strict equal https://developer.mozilla.org/"